ActivityListener/EventTypes.cs

```diff
@@ -5,10 +5,12 @@ public static class EventTypes
         public const string PersonCreatedEvent = "PersonCreatedEvent";
         public const string PersonUpdatedEvent = "PersonUpdatedEvent";
 
+        public const string PersonAddedToTenureEvent = "PersonAddedToTenureEvent";
+
         public const string ContactDetailAddedEvent = "ContactDetailAddedEvent";
         public const string ContactDetailDeletedEvent = "ContactDetailDeletedEvent";
 
         public const string TenureCreatedEvent = "TenureCreatedEvent";
-        public const string PersonAddedToTenureEvent = "PersonAddedToTenureEvent";
```

Comment:
Unless I've missed something, I don't think this should be removed as this is an event the the other patch endpoint in Tenure API

Reply:
I didn't remove it, I just moved it to line 8.

Reply:
oh sorry my mistake, I didn't see that part